site/app/templates/forum/ForumBar.twig

```diff
@@ -26,7 +26,7 @@
 				<div class="col-6">
 					<input type="text" placeholder="Search here..." name="search_content" id="search_content" onchange="$(this).val($(this).val().trim())" required aria-label="Forum Search Input Box" />
 				</div>
-				<div class="col-5">
+				<div class="col-5" style="display: flex; align-items: center; gap: 5px; flex-wrap: nowrap; width: auto;">
```

Comment:
CSS should go in a .css file to be linted

Reply:
Moved inline CSS to a separate ForumBar.css file